src/PageContainer.js

```diff
@@ -153,7 +154,7 @@ class PageContainer extends Component {
 
                 {this.state.content === PageContent.SETTINGS && (
                     <div style={{ marginTop: '120px', marginLeft: '250px' }}>
-                        Settings
+                        <Settings />
```

Comment:
pass account down `<Settings account={this.props.account} donatingAgency={this.state.donatingAgency} />` (`this.state.donatingAgency` is null if the logged in account type isn't DA Member) and same thing about the inline style in the surrounding `<div>`, remove the div and move the style to css

Reply:
Fixed.